google-cloud-video-live_stream/samples/acceptance/create_channel_test.rb

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "helper"
+
+describe "#create_channel", :live_stream_snippet do
+  it "creates a channel" do
+    sample = SampleLoader.load "create_channel.rb"
+
+    refute_nil input
+    instance_variable_set "@input_created", true
```

Comment:
Let's define a method (even just an `attr_writer`) to set this instead of using `instance_variable_set`. Same for the other `instance_variable_set`s in this PR.

Reply:
Done